MS3/K013.mscx

```diff
@@ -12519,7 +12519,7 @@
             </Chord>
           <Harmony>
             <harmonyType>1</harmonyType>
-            <name>I.#viio/iv/vi{</name>
+            <name>vi.#viio/iv{</name>
```

Comment:
m. 71-78: It is a procedural error to not mention such a profound change in the commit message. You have decided to re-interpret the localkey structure of the passage, which goes beyond merely eliminating warnings. By turning secondary keys into local keys you are eliminating information on the hierarchical levels and that would have to be reviewed by the original annotator (and/or reviewer).